tests/ms2deepscore/test_ms2deepscore_integration.py

```diff
@@ -6,20 +6,23 @@
 def test_match_spectra_from_path_with_2_spectra(
     mgf_path_of_2_spectra, ms2deepscore_client
 ):
+    n_best = 3
     result = ms2deepscore_client.load_and_match_spectra(
```

Comment:
when we deploy new version of client this test needs to be updated with the new method name, let's not forget this. Maybe add a TODO?

Reply:
It should be updated now already. Probably test is not passing

Reply:
Yeah this one isn't tested yet